src/documentation/kv1_read.md

```diff
@@ -0,0 +1,125 @@
+stop_lon |  |  | The coordinates of the stop_area are computed as the barycentre of all the associated stop_points.
+location_type |  |  | Fixed value `1`.
+
+### lines.txt
+NTFS field | KV1 file | KV1 field | Mapping rule/Comment
+--- | --- | --- | ---
+line_id | LINEXXXXXX.TMI | *LinePlanningNumber* | This field is prefixed.
+line_code | LINEXXXXXX.TMI | *LinePublicNumber* | 
+line_name |  |  | This field is computed using the the name of the assoicated Route in the forward direction (identical to forward_line_name).
+forward_line_name |  |  | This field is computed using the the name of the assoicated Route in the forward direction.
+forward_direction |  |  | This field is computed using the last stop_area of the associated Route in the forward direction.
+backward_line_name |  |  | This field is computed using the the name of the assoicated Route in the backward direction.
+backward_direction |  |  | This field is computed using the last stop_area of the associated Route in the backward direction.
+line_color | LINEXXXXXX.TMI | *LineColor* | 
+network_id | LINEXXXXXX.TMI | *DataOwnerCode* | This field is prefixed. Link to the file [networks.txt](#networkstxt).
+commercial_mode_id | LINEXXXXXX.TMI | *TransportType* | This field is not prefixed. Link to the file [commercial_modes.txt](#comercialmodestxt).
+
+### routes.txt
+A Route is created by a line and a direction stated in the JOPAXXXXXX.TMI file.
+
+NTFS field | KV1 file | KV1 field | Mapping rule/Comment
+--- | --- | --- | ---
+route_id | JOPAXXXXXX.TMI | *LinePlanningNumber*, *Direction* | This field is prefixed. Concatenation of *LinePlanningNumber* and *Direction* separated by a `:`. Ex: "\<prefix\>:2029:2"
+route_name | JOPAXXXXXX.TMI |  | "[most frequent first stop of contained trips] - [most frequent last stop of contained trips]"
+direction_type | JOPAXXXXXX.TMI | *Direction* | `forward` value when *Direction* is `1` or `A`. `backward` in all other cases.
+line_id | JOPAXXXXXX.TMI | *LinePlanningNumber* | This field is prefixed. Link to the file [lines.txt](#linestxt).
+destination_id |  |  | This field is computed using the most frequent last stop_area of contained trips.
+
+### calendar_dates.txt
+NTFS field | KV1 file | KV1 field | Mapping rule/Comment
+--- | --- | --- | ---
+service_id | OPERDAYXXX.TMI | *OrganizationalUnitCode*, *ScheduleCode*, *ScheduleTypeCode* | This field is prefixed. Concatenation of the 3 specified fields separated by a ':'. Ex: "\<prefix\>:2029:1:1"
+date | OPERDAYXXX.TMI |  | Service date to be transformed into the YYYYMMDD format.
+exception_type |  |  | Fixed value `1`.
+
+### trips.txt
+NTFS field | KV1 file | KV1 field | Mapping rule/Comment
+--- | --- | --- | ---
+route_id | JOPAXXXXXX.TMI | *LinePlanningNumber*, *Direction* | This field is prefixed. Link to the files [routes.txt](#routestxt). Concatenation of *LinePlanningNumber* and *Direction* separated by a `:`. Ex: "\<prefix\>:2029:2"
```

Comment:
```suggestion
route_id | JOPAXXXXXX.TMI | *LinePlanningNumber*, *Direction* | Linked by *JourneyPatternCode* in `PUJOPASSXX.TMI` and `JOPAXXXXXX.TMI`. This field is prefixed. Link to the files [routes.txt](#routestxt). Concatenation of *LinePlanningNumber* and *Direction* separated by a `:`. Ex: "\<prefix\>:2029:2"
```

Reply:
I don't understand this comment. The `route_id` is only defined by *LinePlanningNumber* and *Direction*. Can you explain me the need for this addition ?

Reply:
PUJOPASSXX.TMI does not have the field `Direction`

Reply:
trips are in `PUJOPASSXX.TMI` and I need to get `route_id` from `JOPAXXXXXX.TMI` so I need a link beetween `PUJOPASSXX.TMI` and `JOPAXXXXXX.TMI` which is `JourneyPatternCode`

Reply:
I've included a notice before the mapping array. I think it's better this way